controllers/home.php

```diff
@@ -3,5 +3,6 @@
 declare(strict_types=1);
 
 $ads = (new \App\Ads())->getAds();
+$branches = (new \App\Branch())->getBranches();
 
-loadView('home', ['ads' => $ads]);
\ No newline at end of file
+loadView('home', ['ads' => $ads, 'branches' => $branches]);
```

Comment:
Could be refactored like this:
loadView('home', [
'ads'      => (new \App\Ads())->getAds(),
'branches' => (new \App\Branch())->getBranches()
]);